Sources/SwiftParser/Declarations.swift

```diff
@@ -1718,6 +1718,7 @@ extension Parser {
 
     var fixity: RawTokenSyntax?
     var leftInFixityModifier: RawUnexpectedNodesSyntax?
+    var modifiersAfterFixity: RawUnexpectedNodesSyntax?
```

Comment:
I don’t think these need to be separate variables. We can just combine the in the `if`-branch already. And I think variable-name wise, `afterFixity` would be a good name to match `beforeFixity`.